<launch>
  <arg name="train_data" default="train_data" />
  <arg name="train_type" default="publish_spectrogram" />
  <arg name="target_class" default="test" />
  <node pkg="sound_classification" type="hit_spec_axis.py" name="cut_spectrogram" output="screen" respawn="true">
    <param name="target_class" value="$(arg target_class)"/>
    <param name="train_data" value="$(arg train_data)" />
    <remap from="~hit_spectrogram" to="/$(arg train_type)/hit_spectrogram" />
    <remap from="~cut_spectrogram" to="/$(arg train_type)/cut_spectrogram" />
  </node>

</launch>
